Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs

```diff
@@ -39,7 +39,7 @@ public MapRenderer() : base()
                 new PolylineLogic(),
                 new PolygonLogic(),
                 new CircleLogic(),
-                new PinLogic(),
+                new PinLogic(OnMarkerCreating, OnMarkerCreated, OnMarkerDeleting, OnMarkerDeleted),
```

Comment:
It will be greate to Add same functionality for iOS as well.

Reply:
Sure, I will add to iOS side too! 👍